### PR TITLE
decrease token metadata update interval 

### PR DESCRIPTION
**Pull request by ayrat555**
fixes https://github.com/poanetwork/blockscout/issues/1689

## Motivation

Update interval for token metadata is one week. The app is being deployed more often recently so token metadata cannot be updated.

## Changelog

- decrease token metadata update interval to 2 days

### CODE REVIEW

**Comment by goodsoft** [comment]
Triggering supply update when we encounter a token transfer in indexer seems to be a better idea to me

**Comment by ayrat555** [comment]
I can add it but we still need fetching in the controller because there are already a lot stale tokens

**Comment by goodsoft** [comment]
If I understand correctly, the complete update of all of them should be triggered by application server restart anyway.

**Comment by ayrat555** [comment]
no, it'll be 2 days after the restart

**Comment by goodsoft** [comment]
It [sends a message](https://github.com/poanetwork/blockscout/blob/master/apps/indexer/lib/indexer/token/metadata_updater.ex#L18) to itself immediately.

My point is, we should strive to import blocks with most of relevant metadata in near-realtime on a steady and predictable pace. This way we won't need to add ad-hoc out-of-band requests from frontend to indexer.

The problem with such requests is that they create an attack vector by exposing indexer to outer world and allowing external users to influence its request rate.

File: apps/indexer/test/indexer/token/fetcher_test.exs [hunk 1]
```diff
@@ -13,19 +13,13 @@ defmodule Indexer.Token.FetcherTest do
   describe "init/3" do
     test "returns uncataloged tokens", %{json_rpc_named_arguments: json_rpc_named_arguments} do
       insert(:token, cataloged: true)
-      %Token{contract_address_hash: uncatalog_address} = insert(:token, cataloged: false)
+      %Token{contract_address_hash: uncatalog_address} = insert(:token, cataloged: false) |> IO.inspect()
```
Comment (goodsoft) [inline apps/indexer/test/indexer/token/fetcher_test.exs]: IO.inspect?

Comment (ayrat555) [reply apps/indexer/test/indexer/token/fetcher_test.exs]: my bad

**Pull request merged**